pipeline/pipeline.go

```diff
@@ -0,0 +1,32 @@
+package pipeline
+
+import (
+	"os"
+	"os/exec"
+
+	csvParser "github.com/m-lab/clinic2019/CsvParser"
+)
+
+var BUCKET_NAME = "incidents-location-hierarchy"
+var INCIDENT_CSV = "incidents.csv"
+
+func findIncidents() {
+	runSignalSearcher := "go run github.com/m-lab/signal-searcher  | sort -nk1 > " + INCIDENT_CSV
+	cmd := exec.Command("bash", "-c", runSignalSearcher)
+	cmd.Run()
+}
+
+func runPipeline() {
+	// Generate a CSV file of incidents
+	findIncidents()
+
+	// Temporary place a directory of incidents on the user's desktop to copy files to GCS
+	usersPath := os.Getenv("HOME")
+	incidentPath := usersPath + "/Desktop/generatedIncidents/"
```

Comment:
May be I don't have the right understand of the pipeline but why is this hard coded? Like what is I want/have a different set up on my disk

Reply:
You are totally right, I changed it to be a global variable so that the option for configuration might be more obvious